### DISCUSSION TITLE
Collection Views

### DISCUSSION THREAD

**Discussion by mabeyj** [TriliumNext/Trilium#1896] [Show and tell]
Hi,

I've made an extension that implements Notion-like database views using Render Notes. Specify a search query which is then executed using Trilium's search engine. The results are presented in either a table, Kanban board, or gallery (grid) view. Note attributes can be displayed and formatted in various ways.

Documentation, screenshots, and downloads: https://github.com/mabeyj/trilium-collection-views

**Comment by zadam** [comment]
Wow, that's pretty cool!

**Comment by chee** [comment]
this is amazing

**Reply by ecodiv** [reply]
Great extension, thanks!
